src/App/Layout.tsx

```diff
@@ -14,10 +15,23 @@ const { HEADER_LINKS, GROUPS_DATA, SOCIAL_MEDIA_LINKS } = IS_AST
 
 export default function Layout() {
   return (
-    <div className="grid grid-rows-[auto_1fr_auto]">
+    <div className="grid grid-rows-[auto_auto_1fr_auto]">
```

Comment:
We should immediately remove these banner changes in a followup PR, since we'll want to remove it when `RC-v1.9` gets merged into `master`.